packages/yoroi-extension/app/ergo-connector/utils/connectorMessenger.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import type {
+    ConfirmedSignData,
+    ConnectedSites,
+    ConnectingMessage,
+    ConnectRetrieveData,
+    FailedSignData,
+    GetConnectedSitesData,
+    Protocol,
+    RemoveWalletFromWhitelistData,
+    SigningMessage,
+    TxSignWindowRetrieveData,
+    GetUtxosRequest,
+    ConnectResponseData,
+    GetConnectionProtocolData,
+} from '../../../chrome/extension/ergo-connector/types';
+import type { IGetAllUtxosResponse } from '../../api/ada/lib/storage/models/PublicDeriver/interfaces';
+
+export class ConnectorMessenger {
+    initedConnecting: boolean;
+    initedSigning: boolean;
+
+    constructor() {
+        this.initedConnecting = false
+        this.initedSigning = false
+    }
+
+    _sendMessage(
+        message: (
+            ConnectResponseData
+            | ConfirmedSignData
+            | FailedSignData
+            | TxSignWindowRetrieveData
+            | ConnectRetrieveData
+            | RemoveWalletFromWhitelistData
+            | GetConnectedSitesData
+            | GetConnectionProtocolData
+            | GetUtxosRequest
+        )
+    ): Promise<any> {
+        return new Promise((resolve, reject) => {
+            window.chrome.runtime.sendMessage(
+             message,
+              response => {
```

Comment:
Please fix inconsistent indentations of this line and the previous.

Please use 2 space indentations for this file for consistency.